go.mod

```diff
@@ -74,6 +74,8 @@ require (
 	github.com/fatih/structs v1.1.0
 )
 
+require github.com/yusufpapurcu/wmi v1.2.2 // indirect
```

Comment:
why move this?

Reply:
no special reason, shown in two places after merge, removed one